src/PrestaShopBundle/Form/Validator/Constraints/TinyMceMaxLengthValidator.php

```diff
@@ -27,25 +27,44 @@
 namespace PrestaShopBundle\Form\Validator\Constraints;
 
 use PrestaShop\PrestaShop\Adapter\LegacyContext;
+use PrestaShop\PrestaShop\Adapter\Validate;
 use Symfony\Component\Validator\Constraint;
 use Symfony\Component\Validator\ConstraintValidator;
+use Symfony\Component\Validator\Exception\UnexpectedTypeException;
 
 class TinyMceMaxLengthValidator extends ConstraintValidator
 {
     public function validate($value, Constraint $constraint)
     {
+        if (!$constraint instanceof TinyMceMaxLength) {
+            throw new UnexpectedTypeException($constraint, TinyMceMaxLength::class);
+        }
+
+        if (!Validate::isUnsignedInt($constraint->max)) {
```

Comment:
Inject the validator in the constructor instead

Reply:
Hi @PierreRambaud ,
Thanks, can you please explain why ?

Reply:
It's because the Validate class shouldn't contain static method.
Static methods (a.k.a singleton) are hard to test, even if it looks easier to use, it's a bad practice, using dependency injection is the right way to do it :)

Reply:
Thanks @PierreRambaud ... Done !